### ISSUE TITLE
[CAIA Intake] Draft a standard email that can be sent to assistive tech participants to inform them of available VA blind rehab resources

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#90370]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name**: CAIA AT research support
- **OCTO product owner:** @artsymartha68 
- **Product name:** AT research follow up email
- **Product manager:**  @artsymartha68 
- **Slack channel:** CAIA channel
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** @sara-amanda 
 
## About your initiative

During AT research sessions, participants often don't know their tech and all it can do for them. Researchers and a11y specialists report that they would like to help if possible and don't want to stress anyone out during a session. 

I talked with a couple of our Blind Rehab Services folks and they recommended the following:

- Tell Veterans to reach out to their local VIST coordinator, or contact one if they don't have one
- "Visual Impairment Services Team (VIST) Coordinator services are available at all VHA Medical Centers. Please contact your local VIST Coordinator or speak to your local Primary Care Provider for additional information."
- [This PDF](https://www.rehab.va.gov/PROSTHETICS/factsheet/BRS-FactSheet-VisualImpairmentServicesTeam.pdf) has details we can pull and also link to 
- Go to the main [blind rehab web page](https://www.rehab.va.gov/blindrehab/) 
- Virtual visits may be available if traveling to a VA location isn't possible
- It doesn't have to be a long message. The main take away should be encouraging, many blind/low vision Veterans may not be aware that these services are available. 

Need to coordinate with @sstrassberg as to how this gets sent out via Perigean folks. 

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ x ] Something else:  I'm envisioning a standard email that we can send to participants with a reminder of the training and other resources that are available to them

### What's the nature of your initiative and desired outcomes?

I envision a standard email that can be sent after an AT research session to remind the Veteran that services and training are available.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

No particular timeline

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [ ]  Yes
- [x]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** `Enter dates`
- **What is the name of your engineer who will build the React widget?** `Name`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact<name>]

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Naomi Eke`, and we’ll then acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

**Comment by katherine-fung** [comment]
Noting that before we start this work, we want to ask Danielle in our sync next week (8/21) if there's guidance or a precedent for this type of email to research testers.

**Comment by katherine-fung** [comment]
@artsymartha68 when you say "Virtual visits may be available if traveling to a VA location isn't possible," do you mean virtual visits with a VIST coordinator may be available? Or virtual visits with health care providers?

**Comment by artsymartha68** [comment]
Probably both are true but let's focus on VIST for this purpose.

**Comment by katherine-fung** [comment]
Hi @artsymartha68, 

A and I worked on this email copy. Let me know if you have any questions or feedback. Thanks! 

https://docs.google.com/document/d/1FHRXYxXS8A9hXgCGGJRjhgl-GeTntFB90ylD7Y_6p4E/edit

**Comment by artsymartha68** [comment]
This is wonderful! Thanks so much for taking the time to write this up.

**Comment by artsymartha68** [comment]
Yes! Have you sent it along to Shane for sending out after AT sessions?

**Comment by artsymartha68** [comment]
Closing this issue, sent the email template to @sstrassberg to forward onto to Perigean.

**Comment by sara-amanda** [comment]
## OCTO Research-Ops Lead, Shane Strassberg
- [Profile for Shane](https://dsva.slack.com/team/U01RMUH860G)
-  **GitHub**: [sstrassberg](https://github.com/sstrassberg)


### OCTO Outreach 

[9/17/2024 Chat via Research-Ops Channel in OCTO Slack via Martha](https://dsva.slack.com/archives/C0216PL32HJ/p1726576227216509) 

> **Martha Wilkes**
> [@Shane Strassberg](https://dsva.slack.com/team/U01RMUH860G)
>  The CAIA folks [drafted an email](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90370) per my request to send to AT participants after their sessions. We often see folks struggling with new AT, and we want to remind them that VA training services are available to them. Is it possible to add it to the standard operating procedure after these sessions end?
> cc: [@Lily Strelich (OOO 9/23-27, 10/1-7)](https://dsva.slack.com/team/U05LYEK4ZEH), [@Katherine Fung](https://dsva.slack.com/team/U03FLTCRC57), [@A Procik](https://dsva.slack.com/team/U05NW7HGCE5)


### Reply from Shane

> **Shane Strassberg**
> [@Martha Wilkes](https://dsva.slack.com/team/UNAC3RND9)
>  Shouldn't be a problem. I can't open the google doc on my GFE, can it be sent as a standard word doc to my email so I can forward it on to Perigean?

### Next Steps

- [ ] Word Doc needed for Shane c/o Martha
- [ ] Shane to send on to Perigean

cc: @artsymartha68 @strelichl @NaomiPMC

**Comment by artsymartha68** [comment]
@sstrassberg the email is finalized and ready to be sent to any assistive tech participants. Will you coordinate that with Perigean? Let me know how I can help!
